pack/eoe/eoep.json

```diff
@@ -890,6 +941,21 @@
         "type_code": "skill",
         "xp": 1
     },
+    {
+        "code": "08080",
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Marcin Jakubowski",
+        "name": "Fire Extinguisher",
+        "pack_code": "eoep",
+        "position": 80,
+        "quantity": 2,
+        "skill_agility": 2,
+        "slot": "Hand",
+        "text": "[action]: <b>Fight.</b> You get +1 [combat] and deal +1 damage for this attack.\n[action] Discard Fire Extinguisher: <b>Evade.</b> Automatically evade all enemies engaged with you. You may exile Fire Extinguisher instead of discarding it to discard each non-<b><i>Elite</i></b> evaded by this effect.",
```

Comment:
I believe we use [[Elite]] instead (and for anything that is a Trait in the card text)